src/lib/components/LivestreamContainer/index.js

```diff
@@ -37,24 +38,45 @@ class LivestreamContainer extends BaseStateElement {
       </div>
 
       <div class="web-livestream-container__col-chat">
-        <iframe
-          class="w-youtube-chat"
-          src="https://www.youtube.com/live_chat?v=${this.videoId}&amp;embed_domain=${location.hostname}"
-          frameborder="0"
-        ></iframe>
+        ${this.isChatActive ?
+          html`
+            <iframe
+              class="w-youtube-chat"
+              src="https://www.youtube.com/live_chat?v=${this.videoId}&amp;embed_domain=${location.hostname}"
+              frameborder="0"
+            ></iframe>
+          ` :
+          html`
+            <div class="w-youtube-disabled-chat">
+              <div class="w-youtube-disabled-chat__container">
+                <div>
+                  Live Chat is currently disabled. Please head to YouTube and
+                  ask your questions in the comments on the video.
+                </div>
+                <a
+                  class="w-youtube-disabled-chat__link" 
+                  href="https://www.youtube.com/watch?v=${this.videoId}">
+                  Go to YouTube
+                </a>
+              </div>
+            </div>
+          `
+        }
       </div>
     `;
   }
 
   /**
    * @param {!Object<string, *>} state
    */
-  onStateChanged() {
-    // Example implementation depending on what we end up putting in our
-    // state object:
-    // const {videoId, isChatEnabled} = state;
-    // this.videoId = videoId;
-    // this.isChatEnabled = isChatEnabled;
+  onStateChanged({activeEventDay}) {
+    if (!activeEventDay) {
+      return;
```

Comment:
Should this reset the state of this element? If for some reason our logic is bad and `activeEventDay` transitions back to `null`, then we won't clear state.

But I think `EventStore` will only ever go from `null` to a value, not the other way around.

Reply:
> Should this reset the state of this element? If for some reason our logic is bad and activeEventDay transitions back to null, then we won't clear state.

If it transitions to null then I think we'd want to keep the previous video on screen. Otherwise the player would show a blank rectangle.